.github/workflows/publish-sdist-wheels.yml

```diff
@@ -62,7 +62,12 @@ jobs:
         run: |
           # used by setup.py to decide if to set `FindHDF5` to use static hdf5 libraries
           export STATIC_HDF5=True
-          export CMAKE_PREFIX_PATH=/opt/hdf5-static/install-x86_64/install/
+
+          if [[ "$(uname -m)" == "aarch64" ]]; then
+            export CMAKE_PREFIX_PATH=/opt/hdf5-static/install-aarch64/install/
```

Comment:
what about `export CMAKE_PREFIX_PATH="/opt/hdf5-static/install-$(uname -m)/install/"`?

Reply:
Good point! Changed it.

Reply:
sorry I missed it before.